packages/grafana-ui/src/components/Button/Button.story.tsx

```diff
@@ -26,11 +30,53 @@ export default {
   },
 };
 
-export const Simple: Story<ButtonProps> = ({ children, ...args }) => <Button {...args}>{children}</Button>;
-Simple.args = {
-  variant: 'primary',
-  size: 'md',
-  disabled: false,
-  children: 'Button',
-  icon: undefined,
+export const Variants: Story<ButtonProps> = ({ children, ...args }) => {
```

Comment:
I think the point of the story is to provide controls for the button and then allow users to modify them to be able to see different variants. Removing controls and showing only some variants for the buttons instead is quite limited and doesn't showcase all the possible options. I think a better place for this Variants would be in the docs, i.e. the mdx file where we can show different variants and add a bit of description of where and how they'd be used. The canvas on the other hand should have mostly one component, which can then be modified via controls.

Reply:
I find this type of story really bad for development & for developers to see examples. I find showing all varaiants at the same time is so much better for development and to showcase examples.

Reply:
But we can achieve exactly the same showcase by moving the components to the docs tab while keeping canvas as a sort of playground to try out various options.

Reply:
the playground was just changing icon, not that valuable a playground. But yea a separate playground could be added if that is useful. But I prefer the canvas showing the important state for easy development & verification